rcl/include/rcl/types.h

```diff
@@ -101,7 +101,7 @@ typedef rmw_ret_t rcl_ret_t;
 /// Argument is not a valid parameter rule
 #define RCL_RET_INVALID_PARAM_RULE 1010
 /// Argument is not a valid log level rule
-#define RCL_RET_INVALID_LOG_LEVEL 1020
+#define RCL_RET_INVALID_LOG_LEVEL_RULE 1020
```

Comment:
Do we need to revert this change too? I thought we essentially just wanted users to be able to provide the double-underscore versions; not sure that we need to maintain this particular API compatibility.
Alternatively, we can not compromise the new API by adding both the new and old definitions.

Reply:
I don't have a strong opinion on this, mainly because these return codes are not being used by the public API -- which raises the question as to why these are exposed in the first place. All `RCL_RET_*_RULE` return codes are used internally only. 

The sole reason why `RCL_RET_INVALID_LOG_LEVEL_RULE` was switched to `RCL_RET_INVALID_LOG_LEVEL` is that the former no longer made sense with the rule-like arguments gone.

Moving forward, we can (1) go back and find another return code for the now deprecated log level rule parsing function, (2) stay as-is, as there's no actual need for a special return code or (3) keep both but outside our public API.

Reply:
If it's not necessary to be public, then I'm fine leaving it as-is (returning RCL_RET_ERROR).